java-jbang/src/main.java

```diff
@@ -0,0 +1,12 @@
+//DEPS com.pulumi:pulumi:0.1.0
```

Comment:
Could we make it a floating ref?

Reply:
Perhaps `//DEPS com.pulumi:pulumi:0.+` - this seems to work for me.

Reply:
yes, that syntax kinda works. probably will add "(,1.0]" syntax in future.